rvip/README.md

```diff
@@ -0,0 +1,141 @@
+# Robotic Vision Integration Package (RVIP) Documentation
```

Comment:
I'm somewhat confused: quite a bit of content in this file seems duplicated in the main `readme.md` and `demo.md`?

Reply:
This `README.md` file is used for the automatic generation of **rvip** documentation by **rosdoc_lite** and allow users to set **rvip** up as **as a generic solution**. 

`readme.md` is to show the documentation up-front on GitHub and allow users to set **rvip** up **as a generic solution**.

`demo.md` is to allow users to setup **rvip** up using the recommended specific external dependencies **as a complete robotic vision suite**; Instructions here are for users who have no idea what external dependencies they need but wish to quickly use the robotic vision features **rvip** provides.